tests/test_input.py

```diff
@@ -0,0 +1,323 @@
+                    "type": "pointer",
+                    "id": "main",
+                    "actions": [{
+                        "type": "pointerDown",
+                        "button": 0,
+                    }, {
+                        "type": "pointerMove",
+                        "x": 1,
+                        "y": 1,
+                    }, {
+                        "type": "pointerUp",
+                        "button": 0,
+                    }]
+                }]
+            }
+        })
+
+    result = await execute_command(
+        websocket, {
+            "method": "script.evaluate",
+            "params": {
+                "expression": "allEvents",
+                "awaitPromise": False,
+                "resultOwnership": "root",
+                "target": {
+                    "context": context_id
+                }
+            }
+        })
+
+    assert {
+        'type': 'success',
+        'result': {
+            'type': 'array',
+            'value': [{
+                'type': 'object',
+                'value': [['event', {
+                    'type': 'string',
+                    'value': 'mousedown'
+                }], ['button', {
+                    'type': 'number',
+                    'value': 0
+                }], ['buttons', {
+                    'type': 'number',
+                    'value': 1
+                }], ['clientX', {
+                    'type': 'number',
+                    'value': 0
+                }], ['clientY', {
+                    'type': 'number',
+                    'value': 0
+                }]]
+            }, {
+                'type': 'object',
+                'value': [['event', {
+                    'type': 'string',
+                    'value': 'mousemove'
+                }], ['button', {
+                    'type': 'number',
+                    'value': 0
+                }], ['buttons', {
+                    'type': 'number',
+                    'value': 1
+                }], ['clientX', {
+                    'type': 'number',
+                    'value': 1
+                }], ['clientY', {
+                    'type': 'number',
+                    'value': 1
+                }]]
+            }, {
+                'type': 'object',
+                'value': [['event', {
+                    'type': 'string',
+                    'value': 'mouseup'
+                }], ['button', {
+                    'type': 'number',
+                    'value': 0
+                }], ['buttons', {
+                    'type': 'number',
+                    'value': 0
+                }], ['clientX', {
+                    'type': 'number',
+                    'value': 1
+                }], ['clientY', {
+                    'type': 'number',
+                    'value': 1
+                }]]
+            }],
+            'handle': ANY_STR
+        },
+        'realm': ANY_STR
+    } == result
+
+
+# TODO(jrandolf): Fix scroll
```

Comment:
what's with the scroll?

Reply:
Scroll doesn't work by itself without some mouse input. I've updated the test to showcase this.

Reply:
Yea, the scroll is flaky. Probably because we don't use pixel precise wheel movements in CDP. I'm going to comment this test out.

Reply:
scroll is on a different thread in the renderer, so that could be a reason for flakiness

Reply:
That too. Scroll has a lot of problems :P